app/styles/nypr-ui/_story.scss

```diff
@@ -0,0 +1,55 @@
+article figure {
+  margin-bottom: 30px;
+}
+
+article footer {
+  max-width: 630px;
+  margin-left: auto;
+  margin-right: auto;
+
+  &.text {
+    color: $lightgray;
+  }
+}
+
+article section {
```

Comment:
Sure! You're thinking in case we add custom HTML build out inside story pages. 👍